Maple2.Server.Game/Manager/Items/InventoryManager.cs

```diff
@@ -154,20 +154,20 @@ public class InventoryManager {
                 return false;
             }
 
+            if (!CanAdd(add)) {
```

Comment:
I would prefer just checking `OpenSlots` in conjunction with the check below because `CanAdd` is checking multiple things which are already being checked. (e.g. ItemCollection above, and GetStackResult below)